.i18nrc.json

```diff
@@ -25,6 +26,11 @@
     "src/ui/ui_render/bootstrap/app_bootstrap.js",
     "src/ui/ui_render/ui_render_mixin.js",
     "x-pack/plugins/monitoring/public/components/cluster/overview/alerts_panel.js",
-    "x-pack/plugins/monitoring/public/directives/alerts/index.js"
+    "x-pack/plugins/monitoring/public/directives/alerts/index.js",
```

Comment:
PR #25112 is merged, I believe we can remove this exclusions.

Reply:
Yes, we can, except two last exclusions